### PR TITLE
[common-lisp/es-es] Add Spanish translation for common lisp

### PR DESCRIPTION
**Pull request by ivanchoff**
…dded

- [x] I solemnly swear that this is all original content of which I am the original author
- [x] Pull request title is prepended with `[language/lang-code]`
- [x] Pull request touches only one file (or a set of logically related files with similar changes made)
- [x] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [x] If you've changed any part of the YAML Frontmatter, make sure it is formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [x] Yes, I have double-checked quotes and field names!

### CODE REVIEW

**Comment by divayprakash** [comment]
@ivanchoff Thank you for the contribution and welcome to the LXIYM community!

**Comment by Menelion** [comment]
@divayprakash May I take it for review, please? My Spanish is not perfect, but I usually am able to encounter typos :).

**Comment by divayprakash** [comment]
@Menelion Go right ahead!

File: es-es/common-lips-es.html.markdown [hunk 1]
```diff
@@ -0,0 +1,691 @@
...
+(average-efficiency (make-instance 'canoe :number-of-rowers 15))
+; => 2.7725887
+
+
+;;;-----------------------------------------------------------------------------
+;;; 8. Macros
+;;;-----------------------------------------------------------------------------
+
+;;; las Macros le permiten extender la sintaxis del lenguaje, CL no viene con
+;;; un ciclo WHILE, por lo tanto es facil escribirlo, Si obedecemos nuestros
```
Comment (Menelion) [inline es-es/common-lips-es.html.markdown]: un bucle

File: es-es/common-lips-es.html.markdown [hunk 2]
```diff
@@ -0,0 +1,691 @@
...
+        `(tagbody
+           ,block-name
+           (unless ,condition
+               (go ,done))
+           (progn
+           ,@body)
+           (go ,block-name)
+           ,done)))
+
+;;; revisemos la version de alto nivel para esto:
```
Comment (Menelion) [inline es-es/common-lips-es.html.markdown]: versión

File: es-es/common-lips-es.html.markdown [hunk 3]
```diff
@@ -0,0 +1,691 @@
...
+- [CLiki](http://www.cliki.net/)
+- [common-lisp.net](https://common-lisp.net/)
+- [Awesome Common Lisp](https://github.com/CodyReichert/awesome-cl)
+- [Lisp Lang](http://lisp-lang.org/)
+
+
+## Creditos
+
+Muchas Gracias a la gente de Scheme por proveer un gran punto de inicio
+el cual puede ser movido facilmente a Common Lisp
```
Comment (Menelion) [inline es-es/common-lips-es.html.markdown]: fácilmente

**Pull request closed without merging**